### PR TITLE
update README

### PR DESCRIPTION
**Pull request by nim65s**
spline → curves, ref #4

### CODE REVIEW

**Comment by nim65s** [comment]
@pFernbach : could you update the README's example ? It currently contain:
```python
#importing the libraries of interest
from centroidal_dynamics import * #centroidal dynamics utilities
from spline import * #bezier curves utilities
from bezier_com_traj import * #the actual library
```

Otherwise, I think it would be preferable to have nothing instead of that :)

**Comment by pFernbach** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -77,41 +77,41 @@ to python.
...
+                                 computeCOMTraj, zeroStepCapturability)
 
 
 # create an Equilibrium solver, for a robot of 54 kilos. We linearize the friction cone to four generating rays
 eq = Equilibrium("test", 54., 4)
 
 # Now define some contact points ...
-from numpy import array, asmatrix, matrix
-import numpy as np
+P = array([array([x, y, 0]) for x in [-0.05, 0.05] for y in [-0.1, 0.1]])
```
Comment (nim65s) [inline README.md:92]: `array([[x, y, 0] for x in [-0.05, 0.05] for y in [-0.1, 0.1]])`

File: README.md [hunk 2]
```diff
@@ -77,41 +77,41 @@ to python.
...
 
-P = asmatrix(array([array([x,y,0]) for x in [-0.05,0.05] for y in [-0.1,0.1]]))
 
 #and normals
-z = array([0.,0.,1.])
-N = asmatrix(array([z for _ in range(4)]))
+z = array([0., 0., 1.])
+N = array([z for _ in range(4)])
```
Comment (nim65s) [inline README.md:97]: `N = array([[0., 0., 1.]] * 4) `

File: README.md [hunk 3]
```diff
@@ -129,21 +129,20 @@ tstep = -1.
...
 was feasible
 ```
-res.c_of_t # a bezier curve object describing the com trajectory
+result.c_of_t # a bezier curve object describing the com trajectory
 
 #We can check that the end velocity is indeed zero:
-dc_of_t = res.c_of_t.compute_derivate(1) # computing first derivative
-print np.linalg.norm(dc_of_t(dc_of_t.max()))
+dc_of_t = result.c_of_t.compute_derivate(1) # computing first derivative
+print (np.linalg.norm(dc_of_t(dc_of_t.max())))
```
Comment (nim65s) [inline README.md:145]: no space between `print` & `(`

**Pull request merged**